tests/payment/models/Payment - Credit card - Error Finishing Transaction.model.js

```diff
@@ -0,0 +1,64 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import {
+  fillEmail,
+  getRandomEmail,
+  fillProfile,
+} from '../../../utils/profile-actions'
+import {
+  goToPayment,
+  fillShippingInformation,
+} from '../../../utils/shipping-actions'
+import {
+  completePurchase,
+  payWithCreditCard,
+} from '../../../utils/payment-actions'
+import { ACCOUNT_NAMES } from '../../../utils/constants'
+
+export default function test(account) {
+  describe(`Payment - Credit Card  - Error Finishing Transaction - ${account}`, () => {
+    before(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('with only delivery', () => {
+      const email = getRandomEmail()
+
+      setup({ skus: ['289'], account })
+      fillEmail(email)
+      fillProfile({
+        lastName: 'Denied',
```

Comment:
:joy:

Reply:
No John ~Doe~Denied is allowed to buy here 🙅